python/doc/theory/reliability_sensitivity/taylor_importance_factors.rst

```diff
@@ -7,73 +7,37 @@ The importance factors derived from a Taylor expansion are
 defined to rank the sensitivity of the inputs to
 the output for central dispersion analysis.
 
-Let us denote by :math:`\uX` the input random vector.
-Assume that the marginals of :math:`\uX` are independent.
-Suppose that :math:`Z = h(\ux)` is a real function of the
-input, i.e. :math:`n_Z = 1`.
-Assume that the order 1 Taylor expansion of the function :math:`h`
-at the point :math:`\ux = \muX` is exact, i.e.:
+We denote by :math:`\uX` the input random vector. We assume that the marginals of :math:`\uX` are independent.
 
-.. math::
-    h(\ux) = h(\muX)
-    + \sum_{i=1}^{n_X} \frac{\partial h}{\partial x_i} (\muX) (x_i - \mu_i)
-
-where:
+We note :math:`Y = h(\ux)` with :math:`h: \Rset^d \rightarrow \Rset` a scalar function.
 
--  :math:`\muX` is the mean of the input random vector,
+Refer to :ref:`taylor_expansion_moments` to get details on the expressions of the approximations.
 
--  :math:`\frac{\partial h}{\partial x_i} (\muX)`
-   is the partial derivative of the model :math:`h`
-   with respect to the i-th input variable,
-   evaluated at the point :math:`\ux`.
-
-Therefore the expectation of :math:`Z` is:
+The first order  Taylor expansion of :math:`h` around :math:`\vect{\mu} = \Expect{\uX}` yields to the first order approximation of :math:`\Expect{Y}`:
 
 .. math::
-    \Expect{Z} = h(\muX).
+    \Expect{Y} = h(\vect{\mu}).
 
-The independence of the marginals implies:
+The independence of the marginals implies that:
```

Comment:
Adding a bibliographic reference will be of great value for some readers:
```suggestion
The independence of the marginals implies that (see [daveiga2021]:
```

Reply:
I do not agree... it is not daVeiga who is a reference for that result!!! This result is in any book of initiation in Probabilities.....

Reply:
I agree that (Da Veiga et al, 2022) is not the author of the formula, but that is not necessarily implied when we cite a book as a reference. On the other hand , avoiding to add a reference may imply that the equation is new, which is not. All in all, (Da Veiga et al, 2022) is one of the few textbooks on the topic of sensitivity analysis: adding the reference does not mean that Da Veiga created that equation. Still, this is an informative reference on the topic of S.A., where the equation can be found and explained. This is the information the user is looking for when reading a theory help page. Please insert any reference that you may find suitable.